source/_integrations/sma.markdown

```diff
@@ -14,63 +14,11 @@ ha_platforms:
-| power_l1     | W    | Power for phase 1.   |
-| power_l2     | W    | Power for phase 2.   |
-| power_l3     | W    | Power for phase 3.   |
-| frequency    | Hz   | Grid frequency.   |
-| voltage_l1   | V    | Voltage for phase 1.   |
-| voltage_l2   | V    | Voltage for phase 2.   |
-| voltage_l3   | V    | Voltage for phase 3.   |
-| current_l1   | A    | Current for phase 1.   |
-| current_l2   | A    | Current for phase 2.   |
-| current_l3   | A    | Current for phase 3.   |
-| pv_power_a   | W    | Current power generated by the solar panels (A side).   |
-| pv_power_b   | W    | Current power generated by the solar panels (B side).    |
-| pv_voltage_a | V    | Current voltage generated by the solar panels (A side).   |
-| pv_voltage_b | V    | Current voltage generated by the solar panels (B side).   |
-| pv_current_a | A    | Current amperage generated by the solar panels (A side).   |
-| pv_current_b | A    | Current amperage generated by the solar panels (B side).   |
-| pv_gen_meter | kWh  | Total kWh generated to date.   |
-| total_yield  | kWh  | Total power yield from a solar installation.   |
-| daily_yield  | Wh   | The solar plant's yield for today   |
-| grid_power_supplied | W    | Power supplied.   |
-| grid_power_absorbed | W    | Power absorbed.   |
-| grid_total_yield | kWh  | Total power from the grid.   |
-| grid_total_absorbed | kWh  | Total power supplied to the grid.   |
-| current_consumption | W    | Power that you are currently drawing, depending on your installation it can be a combination of the inverter and the grid.   |
-| total_consumption | kWh  | Total power consumption.   |
-
-### Custom Sensors
-
-The SMA WebConnect module supports a wide variety of sensors, but not all of these have been mapped in the `pysma` library. New sensors can be tested by using YAML configuration. The YAML config is deprecated, nevertheless, an example is provided below. You will need a sensor name, the SMA sensor key and the unit. The path attribute is a custom JMESpath to extract the sensor value and typically not required (advanced use only).
-
-Example:
-
-```yaml
-sensor:
-  - platform: sma
-    host: IP_ADDRESS_OF_DEVICE
-    ssl: true
-    verify_ssl: false
-    password: YOUR_SMA_PASSWORD
-    custom:
-      yesterday_consumption:
-          key: "6400_00543A01"
-          unit: kWh
-          factor: 1000
-```
-
-Feel free to submit additional sensors to be added as standard sensors to the [pysma library](https://github.com/kellerza/pysma/blob/master/pysma/__init__.py#L100).
+The SMA WebConnect module supports a wide variety of sensors, but not all of these have been mapped in the `pysma` library. Current available sensors can be found in the [definitions file](https://github.com/rklomp/pysma/blob/dev/pysma/definitions.py) of `pysma`. Feel free to submit additional sensors to be added as standard sensors to the `pysma` library.
```

Comment:
This is end-user documentation, and we try to make it more accessible.

Sending users to Python sources files to find out what is available for them, is not helping with that.

Please add them to the Home Assistant documentation.